Sources/JExtractSwiftLib/Convenience/String+Extensions.swift

```diff
@@ -22,4 +22,14 @@ extension String {
 
     return "\(f.uppercased())\(String(dropFirst()))"
   }
-}
\ No newline at end of file
+
+  /// Returns whether the string is of the format `isX`
+  var hasJavaBooleanNamingConvention: Bool {
+    guard self.hasPrefix("is"), self.count > 2 else {
```

Comment:
Maybe we need to check that the third character is uppercase as well. Otherwise we may be seeing a “iss()” which isn’t a setter but just a funny named method